src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -262,6 +262,12 @@ namespace Microsoft.Management.Deployment
         PackageCatalogOrigin Origin { get; };
         /// The trust level of the package catalog
         PackageCatalogTrustLevel TrustLevel { get; };
+
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 10)]

```

Comment:
You can't add things to contract 10, it shipped with 1.7.

On a related note, when you add contract 11 at the top of the file, put a comment above it like `// For version 1.9`